modules/cli/src/main/scala/scala/cli/commands/new/New.scala

```diff
@@ -54,5 +56,22 @@ object New extends ScalaCommand[NewOptions] {
         allowExecve = true
       ).waitFor()
 
+    if (exitCode == 0) {
+      val inputs = options.shared.inputs(
+        remainingArgs.remaining,
```

Comment:
Hello @zetashift, thank you for start working on this issue. I think there's an bug in this line, as inputs expects a list of files and directories, yet `zetashift/tlkit.g8` was  passed.

I suggest the easiest solution that would be to pass the `output` directory where the g8 template was created:

```suggestion
        Seq(outputGiter8Path),
```

The Giter8 accepts the following options:
```
  -o, --out <value>        Output directory
```

I propose adding a new option to `NewOptions`, such as `--out`, `--dest`, `-o`, which would allow us to create a template in this directory. This would further enable us to reuse this path and pass it into `options.shared.inputs`

When you run `giter8.Giter8` before in code, you should specify the output directory for template.